netstandard/tools/targets/NETStandard.Library.Trimming.targets

```diff
@@ -0,0 +1,74 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <UsingTask TaskName="TrimFiles" AssemblyFile="$(NETStandardToolsTaskDirectory)NETStandard.Tools.dll" />
+
+  <Target Name="_determineTrimPackageInputs">
+    <ItemGroup Condition="'$(RootPackageReference)' != 'true'">
+      <TrimFilesRootPackages Include="@(PackageReference)"/>
+    </ItemGroup>
+
+    <!-- todo: move this stuff to NETCore.App package -->
+    <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
+      <!-- Native host / CLR for netcoreapp -->
+      <TrimFilesRootFiles Include="coreclr.dll;libcoreclr.dylib;libcoreclr.so" />
+      <TrimFilesRootFiles Include="clrjit.dll;libclrjit.dylib;libclrjit.so" />
+      <TrimFilesRootFiles Include="dotnet.exe;dotnet" />
+      <TrimFilesRootFiles Include="hostfxr.dll;libhostfxr.dylib;libhostfxr.so" />
+      <TrimFilesRootFiles Include="hostpolicy.dll;libhostpolicy.dylib;libhostpolicy.so" />
+
+      <!-- treat platform packages as trimmable -->
+      <TrimmablePackages Include="$(PackageConflictPreferredPackages)" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="TrimFilesOnBuild"
+          DependsOnTargets="_determineTrimPackageInputs"
+          AfterTargets="CoreCompile">
+    <ItemGroup>
+      <_trimOtherRuntimeItems Include="@(_LockFileAssemblies)" Exclude="@(_ConflictPackageFiles)" />
+    </ItemGroup>
+
+    <TrimFiles RootFiles="@(IntermediateAssembly);@(TrimFilesRootFiles)"
+               RootPackages="@(TrimFilesRootPackages)"
+               TrimmablePackages="@(TrimmablePackages)"
+               TrimmableFiles="@(TrimmableFiles)"
+               AssetsFilePath="$(ProjectAssetsFile)"
+               TargetFramework="$(TargetFrameworkMoniker)"
+               RuntimeIdentifier="$(RuntimeIdentifier)"
+               RuntimeItems="@(ReferenceCopyLocalPaths)"
+               OtherRuntimeItems="@(_trimOtherRuntimeItems)"
+               PreferNativeImages="$(TrimFilesPreferNativeImages)">
```

Comment:
When they prefer speed over size.  Currently we have both CoreLib.ni.dll and CoreLib.dll in the package.  Using CoreLib.dll instead of ni.dll shaves 8MB off the app (making Hello World under 10 MB).